### ISSUE TITLE
UKTVPlay not working

### ISSUE THREAD

**Issue by betweenrows** [Shuggy999/m3u#1]
OTT says it can't resolve the URL

**Comment by Shuggy999** [comment]
Yes, it appears that the uktvplay live urls have been taken down, and it is no longer possible to view live on their website.

If you know where to see it online from a url, I can extract the stream and keys for you, but otherwise I am afraid that you can't watch them live anymore. Sorry

**Comment by Shuggy999** [comment]
It seems that the old adress that it used to connect to was a cname to [d3d6djw2di32qi.cloudfront.net](https://www.virustotal.com/gui/domain/live-uktv.simplestreamcdn.com/details), This has since been taken down.

**Comment by betweenrows** [comment]
Thank you for the reply.  That's a shame.  I will keep an eye out and get back to you if I can help in any way.
